config/environments/development.rb

```diff
@@ -35,6 +35,19 @@
   # Raises helpful error messages.
   config.assets.raise_runtime_errors = true
 
+  if ENV['SENDGRID_USERNAME'] && ENV['SENDGRID_PASSWORD']
+    # SMTP / Sendgrid settings
+    ActionMailer::Base.smtp_settings = {
+      address:        'smtp.sendgrid.net',
+      port:           '587',
+      authentication: :plain,
+      user_name:      ENV['SENDGRID_USERNAME'],
+      password:       ENV['SENDGRID_PASSWORD'],
+      domain:         'heroku.com',
+      enable_starttls_auto: true,
+    }
+  end
+
   # Raises error for missing translations
```

Comment:
@kaptron Yeah I knew that they were sending via the server log, but I needed to click on the link in the email to do a full cycle test locally to make sure of correct routing and resetting of the password. If EmailerPreviewer I can do this I can remove this. :+1: